ome/resource_discovery_schema.go

```diff
@@ -0,0 +1,469 @@
+package ome
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/listvalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// DiscoveryJobSchema for discovery job schema.
+func DiscoveryJobSchema() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+
+		"discovery_job_id": schema.Int64Attribute{
```

Comment:
schemas require an "id" field. Does it make sense to rename this to "id"?

Reply:
renamed to id